docs/application/vscode-ext/web.md

```diff
@@ -63,87 +63,160 @@ If you do not have a real device, you can run your application in a virtual envi
 
 This section explains how to use VS Code Extension for Tizen Web to develop applications.
 
+### Change Tizen SDK Path
+
+To change a Tizen SDK path, follow these steps:
+
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Change Tizen SDK Path** and press **Enter**.
+3. In the change notification that appears, perform one of the following:
+    - If you do not want to change the path, click **Keep**.
+    - If you want to change the path, click **Change Path**. In the input field of the Command Palette that appears, type the path and press **Enter**.
+
+      > **Note**
+      >
+      > The default path for:
+      >
+      > Window OS: `c:\tizen-studio`
+      >
+      > Linux OS: `/home/username/tizen-studio`
+
+      ![Change Tizen SDK Path](media/tizen_web24.png)
+
+### Install Tizen Baseline SDK
+
+> **Note**
+>
+> You must install Oracle Java Development Kit (JDK) 8 or OpenJDK 12 for Tizen Studio 3.5 and higher.
+>
+> For more information on download and install, see [Oracle Java Download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+
+To install a Tizen Baseline SDK, follow these steps:
+
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Install Tizen Baseline SDK** and press **Enter**.
+3. In the notification that appears, perform one of the following:
+    - If SDK is already installed, click **YES**, and run **Tizen Web: Change SDK Path**.
+
+      ![install check](media/tizen_web31.png)
```

Comment:
Question. I thought the "SDK" in this sentence is the Tizen Baseline SDK.
However in the image below, the image is checking the installation of "Tizen Studio".
To my understand the image should be fixed. Would you please check this point?

Reply:
Tizen Studio includes the SDK.
(This is because the SDK can also be installed with the CLI.)

Therefore, if you have Tizen Studio already installed, it means that the SDK is installed.
That's not strange.